frontend/src/static/js/v_summary_charts.js

```diff
@@ -260,6 +260,35 @@ window.vSummaryCharts = {
     getGroupTotalContribution(group) {
       return group.reduce((accContribution, user) => accContribution + user.totalCommits, 0);
     },
+
+    getGroupName(group) {
+      switch (this.filterGroupSelection) {
+      case 'groupByRepos':
+        return group[0].repoName;
+      case 'groupByAuthors':
+        return group[0].name;
+      default:
+        return '';
+      }
+    },
+
+    isGroupMerged(groupName) {
```

Comment:
you no longer need this anymore as you can just pass the `group name -> isMerged` map as props from `v_summary` to `v_summary_charts`

Reply:
I tried but it seems that changing the value of a map without reassignment does not trigger the watcher of `v_summary` to merge to repo. I still need to reassign value for the entire map attribute.

Reply:
Hmm how about using `vue watch deep`? https://vuejs.org/v2/api/#watch

Reply:
> Hmm how about using `vue watch deep`?

Hey this works. Thanks so much!